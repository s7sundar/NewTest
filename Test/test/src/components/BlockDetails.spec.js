import React from "react";
import { shallow } from "enzyme";
import BlockDetails from "./BlockDetails";

describe('<BlockDetails />', () => {
    it('test one block deatils', () => {
        const wrapper = shallow(
            <BlockDetails
              data={[{
                attributes:{
                    data:'test'
                }
              }]}
            />
        );
        expect(wrapper).toMatchSnapshot();
        // console.log(wrapper.debug())
    });
});