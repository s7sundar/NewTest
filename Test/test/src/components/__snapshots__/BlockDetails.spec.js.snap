// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockDetails /> test one block deatils 1`] = `ShallowWrapper {}`;
